src/Routing/RouteCollection.php

```diff
@@ -233,7 +234,7 @@ protected function _getNames($url)
         if (isset($url['prefix']) && $url['prefix'] !== false) {
             $prefix = strtolower($url['prefix']);
         }
-        $controller = strtolower($url['controller']);
+        $controller = isset($url['controller']) ? strtolower($url['controller']) : null;
```

Comment:
Is this a place we could use the `??` operator?

Reply:
Not with the extra strtolower() op.

Reply:
Right, we can't.